packages/rsdd/rsdd.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for RSDD"
+description:
+  "OCaml bindings for the Rust Decision Diagram (RSDD) library, generated with ocaml-rs."
+maintainer: ["Matt Wang <mxw@cs.washington.edu>"]
+authors: [
+  "Steven Holtzen <s.holtzen@northeastern.edu>"
+  "Matt Wang <mxw@cs.washington.edu>"
+]
+license: "MIT"
```

Comment:
There does not seem to be a license on the upstream / in the tarball.
Is it possible for you to add it?

Reply:
Thanks!